src/test/java/com/google/api/codegen/testdata/discoveries/bigquery.v2.json.overrides

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
As I've commented before, using overrides for something like this loses the pattern, but I'm okay deferring this.